### PR TITLE
Remove compiler directives

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo-devpack-dotnet/issues/143

### CODE REVIEW

File: src/Neo.Compiler.MSIL/MSIL/Converter.cs [hunk 1]
```diff
@@ -1152,12 +1135,8 @@ private int ConvertCode(ILMethod method, OpCode src, NeoMethod to)
                     break;
 
                 default:
-#if WITHPDB
                     logger.Log("unsupported instruction " + src.code + "\r\n   in: " + to.name + "\r\n");
-                    break;
-#else
                     throw new Exception("unsupported instruction " + src.code + "\r\n   in: " + to.name + "\r\n");
```
Comment (erikzhang) [inline src/Neo.Compiler.MSIL/MSIL/Converter.cs]: This line should be removed too?

Comment (shargon) [reply src/Neo.Compiler.MSIL/MSIL/Converter.cs]: Maybe the log, but not the exception, isn't it? @lightszero

Comment (shargon) [reply src/Neo.Compiler.MSIL/MSIL/Converter.cs]: @lightszero we should throw the exception or only log it?

Comment (shargon) [reply src/Neo.Compiler.MSIL/MSIL/Converter.cs]: For me, it have sense to stop the conversion if one instruction is unsupported.

Comment (lightszero) [reply src/Neo.Compiler.MSIL/MSIL/Converter.cs]: I think should throw for here,skip this could be got more error..

**Comment by cloud8little** [comment]
smoke test passed.

neo-devpack-dotnet: master dda641a4030c0562937391e68336a16b3f7c5950
neo-cli: master 51cd29fbe21abb9e1f17f64e5c6d21bc7decbbb9
neo: master ab4830c
neo-vm: master be2ac36bf35a3033d828e0ba0630d390599c487d
1. build project neo-devpack-dotnet. [Pass]
2. build helloworld smart contract.  [Pass]
3. deploy helloworld contract. [Pass]
4. deploy nep5 contract. [Pass]

**Pull request merged**